jberet-core/src/main/java/org/jberet/job/model/AbstractMerger.java

```diff
@@ -13,30 +13,45 @@
 package org.jberet.job.model;
 
 import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import javax.batch.operations.JobStartException;
 
 import org.jberet._private.BatchMessages;
+import org.jberet.spi.JobXmlResolver;
 import org.jberet.util.BatchUtil;
 
-abstract class AbstractMerger<T extends AbstractJobElement> {
+abstract class AbstractMerger<T extends AbstractJobElement> implements Iterable<JobXmlResolver>{
```

Comment:
Yeah, I wasn't convinced it was right, but it was easy. I'll use the value passed in.